lib/cli.js

```diff
@@ -72,6 +73,7 @@ const EXIT_CODE_ERROR = 2;
  * @property {boolean} [quiet]
  * @property {any} [printConfig]
  * @property {boolean} [fix]
+ * @property {string} [globbyOptions]
```

Comment:
[suggest] A `globbyOptions` value passed to `standalone()` should be an object:

```suggestion
 * @property {Record<string, unknown>} [globbyOptions]
```